logtail/logtail.go

```diff
@@ -375,11 +375,9 @@ func (l *Logger) uploading(ctx context.Context) {
 			switch {
 			case l.zstdEncoder != nil:
```

Comment:
Can we remove this field now? Downstream repos stopped using it I think?

Reply:
Yep, I'll send out a cleanup.